frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/prefill-vm-template-state-update.ts

```diff
@@ -249,6 +249,8 @@ export const prefillVmTemplateUpdater = ({ id, dispatch, getState }: UpdateOptio
     });
     storagesUpdate.unshift(...templateStorages);
   } else {
+    // Create a new storage source for URL, Container and BaseImage Disk sources
```

Comment:
the first line does not have to be here
in this file the second one probably does not have to be here either as this is not a updater per se. It is just called once for one template

Reply:
removed the comment